src/components/victory-bar/helper-methods.js

```diff
@@ -28,7 +25,7 @@ module.exports = {
     const x = (stacked && !categories) ? datum.x :
       this.adjustX(datum, index.seriesIndex, calculatedProps);
     const formatValue = (value, axis) => {
-      return isDate(datum[axis]) ? new Date(value) : value;
+      return datum[axis] instanceof Date ? new Date(value) : value;
```

Comment:
This one is 0 sum on bundle size and arguably more clear as `isDate`
https://github.com/lodash/lodash/blob/4.5.1/lodash.js#L9759-L9761